src/main.lua

```diff
@@ -1164,6 +1164,7 @@ addEventingHandler(
 					{ "balance", "startTimestamp", "endTimestamp" }
 				)
 			end
+			msg.ioEvent:addField("Previous-Operator-Stake", previousStake)
```

Comment:
Interesting... was moving this here the linter's choice?

Reply:
we increment after we initially set the field - and the linter said unused, modified variable